sdk/identity/Azure.Identity.BrokeredAuthentication/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Is this file necessary?

Reply:
I believe it serves to pull shared properties from parent folders, but I'm not sure. I was following the folder structure of our existing project.

Reply:
I have seen it missing in newer projects, so I think if it just does the default, it can be removed.

Reply:
I noticed that including the Directory.Build.props modified my project. It pulled the README.md and CHANGELIST.md into the project without me having to explicitly add item.